searx/templates/simple/preferences.html

```diff
@@ -121,7 +121,7 @@ <h2>{{ _('Preferences') }}</h2>
       {% set engine_id = 'engine_' + search_engine.name|replace(' ', '_') + '__' + categ|replace(' ', '_') %}
       <tr>
         <td class="engine_checkbox">{{ checkbox_onoff(engine_id, (search_engine.name, categ) in disabled_engines) }}</td>
-        <th class="name">{{ search_engine.name }}</th>
+        <th class="name">{% if not search_engine.https_support %}{{ icon('exclamation-sign', 'No HTTPS') }}{% endif %} {{ search_engine.name }}</th>
```

Comment:
It works with `warning` instead of `exclamation-sign`.

See https://github.com/searx/searx/blob/42a194898baf96e15edeb9268fbec5ecd4423c09/searx/static/themes/simple/gruntfile.js#L134

Reply:
Thanks.